spec/tictactoe_spec.rb

```diff
@@ -0,0 +1,120 @@
+require_relative '../lib/game'
+require_relative '../lib/player'
+
+describe Game do
+  game = Game.new
+
+  describe '#input valid' do
+    it 'checks if num 1 is valid input' do
+      expect(game.input_valid?('1')).to eql(true)
+    end
+
+    it 'checks if num 12 is valid input' do
+      expect(game.input_valid?('12')).to eql(false)
+    end
+  end
+
+  describe '#winning possibilities table' do
+    it 'checks if all sets in te are empty' do
```

Comment:
[Optional] I feel as though there are some words missing in the statement on line 18.